docs/source/sox_effects.rst

```diff
@@ -1,35 +1,31 @@
-.. _sox_effects:
+.. py:module:: torchaudio.sox_effects
 
 torchaudio.sox_effects
 ======================
 
-.. py:module:: torchaudio.sox_effects
-
 .. currentmodule:: torchaudio.sox_effects
 
-Resource initialization / shutdown
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-.. autofunction:: init_sox_effects
-
-.. autofunction:: shutdown_sox_effects
-
-Listing supported effects
-~~~~~~~~~~~~~~~~~~~~~~~~~
-
-.. autofunction:: effect_names
+.. minigallery:: torchaudio.sox_effects.apply_effects_tensor torchaudio.sox_effects.apply_effects_file
```

Comment:
this should be moved to the end of the section to stay consistent with the other modules formats. also there is some redundancy in the tutorials with those two functions which results in odd display: https://output.circle-artifacts.com/output/job/bcd411e1-349b-48c0-a4ac-b8f704fec6ef/artifacts/0/docs/sox_effects.html

```suggestion
.. minigallery:: torchaudio.sox_effects.apply_effects_file
```